### ISSUE TITLE
Can't load training set

### ISSUE THREAD

**Issue by BingFong** [jprellberg/isbi2019cancer#1]
I want to know how to load the training images.
what is the correct directory of fold_0, fold_1 and fold_2?

I am receiving errors like:

Reloaded modules: dataset, model, utils
Namespace(batch_size=16, dataroot='data', device='cuda', epochs=6, out='results\\20190814T222329Z.ESZY', seed=1)
Model parameters: 25512945
D:\python\isbi2019cancer-master\dataset.py:78: RuntimeWarning: invalid value encountered in long_scalars
  df.loc[df['class'] == 'hem']['file'].count() / len(df),
D:\python\isbi2019cancer-master\dataset.py:79: RuntimeWarning: invalid value encountered in long_scalars
  df.loc[df['class'] == 'all']['file'].count() / len(df),
Trainset length: 0
Validset length: 1867
class_weights = tensor([nan, nan], device='cuda:0')

Thank you.

**Comment by jprellberg** [comment]
The directory structure is as follows:

```
data/fold_0/all/*.bmp
data/fold_0/hem/*.bmp
data/fold_1/all/*.bmp
data/fold_1/hem/*.bmp
data/fold_2/all/*.bmp
data/fold_2/hem/*.bmp
data/phase2/*.bmp
data/phase3/*.bmp
data/phase2.csv
```

The `fold_0` to `fold_2` directories contain the training data initially released during the challenge. The `phase2` directory contains the preliminary test set data. Later, labels for the preliminary test set have been released, which are saved in the `phase2.csv` file. Finally, test data is inside directory `phase3`.
